Failure looks real:
```
======================================================================
ERROR: test_iteration (corehq.util.tests.test_doc_processor.CaseResumableSqlModelIteratorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/tests/test_doc_processor.py", line 288, in setUp
    super(CaseResumableSqlModelIteratorTest, self).base_setUp()
  File "/mnt/commcare-hq-ro/corehq/util/tests/test_doc_processor.py", line 184, in base_setUp
    self.itr = self.get_iterator()
  File "/mnt/commcare-hq-ro/corehq/util/tests/test_doc_processor.py", line 191, in get_iterator
    return resumable_sql_model_iterator(self.iteration_key, reindex_accessor, chunk_size)
  File "/mnt/commcare-hq-ro/corehq/util/doc_processor/sql.py", line 54, in resumable_sql_model_iterator
    if reindex_accessor.is_sharded():
  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/sql/dbaccessors.py", line 152, in is_sharded
    return isinstance(cls.model_class.objects, RestrictedManager)
AttributeError: 'property' object has no attribute 'objects'
```

--- a/corehq/blobs/tests/test_migrate.py
+++ b/corehq/blobs/tests/test_migrate.py
@@ -549,7 +549,7 @@ class TestMigrateBackend(TestCase):
             return item, ident
         self.sql_docs = []
         with mod.allow_form_processing_queries():
-            for rex in self.sql_reindex_accessors:
+            for rex in (x() for x in self.sql_reindex_accessors):
                 item, ident = create_obj(rex)
                 helper = rex.blob_helper({"_obj_not_json": item})
                 db1.put(StringIO(data), ident, helper._blobdb_bucket())

--- a/corehq/form_processor/backends/sql/dbaccessors.py
+++ b/corehq/form_processor/backends/sql/dbaccessors.py
@@ -141,15 +141,14 @@ class ShardAccessor(object):
 class ReindexAccessor(six.with_metaclass(ABCMeta)):
     startkey_min_value = datetime.min
 
-    @classmethod
-    def is_sharded(cls):
+    def is_sharded(self):
         """
         :return: True the django model is sharded, otherwise false.
         """
         # TODO check for subclass of PartitionedModel when PR merged:
         # https://github.com/dimagi/commcare-hq/pull/14852
         from corehq.form_processor.models import RestrictedManager
-        return isinstance(cls.model_class.objects, RestrictedManager)
+        return isinstance(self.model_class.objects, RestrictedManager)
 
     @abstractproperty
     def model_class(self):

